bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Program.cs

```diff
@@ -152,4 +153,20 @@ void ConfigureAppliationInsightsServices()
 
         logger.LogInformation("Startup // ApplicationInsightsConnectionString = {applicationInsightsConnectionString}", applicationInsightsConnectionString);
     }
+}
+
+async Task ConfigureKeyVaultSettings(ConfigurationManager config)
+{
+    KeyVaultSettings keyVaultSettings = new KeyVaultSettings();
+
+    config.GetSection("KeyVaultSettings").Bind(keyVaultSettings);
```

Comment:
so i think this can be removed as this is done in ProgramConfigurationAndDependencyInjection.cs

Reply:
ok